conjure-lib/src/main/java/com/palantir/conjure/java/lib/Bytes.java

```diff
@@ -105,4 +112,19 @@ public static Bytes from(ByteBuffer buffer) {
 
         return new Bytes(safe);
     }
+
+    static final class Serializer extends JsonSerializer<Bytes> {
+        @Override
+        public void serialize(Bytes value, JsonGenerator gen, SerializerProvider serializers) throws IOException {
+            gen.writeBinary(value.safe);
+        }
+    }
+
+    static final class Deserializer extends JsonDeserializer<Bytes> {
+        @Override
+        public Bytes deserialize(JsonParser parser, DeserializationContext ctxt) throws IOException {
+            // Avoid making a copy of the value from jackson
+            return new Bytes(parser.getBinaryValue());
```

Comment:
We must trust jackson.
This change should fix https://github.com/palantir/conjure-java/pull/158

Reply:
why does this fix #158? (as noted on #158, it's not clear what exactly is failing or how to debug it)

Reply:
It looks like the verifier is testing that `[]` is not valid binary data because our sec requires base-64 encoded strings. Jacksons reflective deserialization allows a json array of bytes to deserialize into `byte[]`. By using JsonParser.getBinaryValue we disallow that codepath.